hr_employee_age/models/hr_employee.py

```diff
@@ -33,9 +32,11 @@ class HrEmployee(models.Model):
     )
 
     @api.one
```

Comment:
`@api.one` is deprecated now (they have decided in this way although here is useful), so better to remove in favor of `@api.multi` and a loop with `for record in self:` and the rest of the stuff replacing `self` by `record`.